src/routes/trade/trade.jsx

```diff
@@ -62,11 +67,35 @@ const Trade = () => {
       );
     }
   };
-
+  const fetchMarketValues = async (symbol, getOHLC) => {
+    await subscribe(
+      {
+        ticks_history: symbol,
+        style: "candles",
+        end: "latest",
+        count: 1000,
+        granularity: 86400,
+        adjust_start_time: 1,
+        subscribe: 1,
+      },
+      getOHLC
+    );
+  }
   onCleanup(() => {
     forgetAll("ticks");
     setMarketTicks({});
   });
+  const getOHLC = (resp) => {
+    const { msg_type, ohlc } = resp;
+    if (msg_type === "ohlc") {
+      setDayLow(parseFloat(ohlc.low));
+      setDayHigh(parseFloat(ohlc.high));
+      setStepValue(ohlc.pip_size);
+    }
+  };
+  const getStatus = (event) => {
```

Comment:
since this function doesn't return any value, the 'getstatus' name doesn't suit this function. and as it is a 1 line function, it can be written as inline function
`<DisplayTick symbol={selectedTradeType()?.symbol} setStatusValue={(status_value} => setStatus(status_value)} />`